anvch.github.io/src/app/blog/page.tsx

```diff
@@ -0,0 +1,27 @@
+import BlogPreview from "../components/blogPreview";
+import blogs from "../blogData";
+
+export default function Blog() {
+  return (
+    <>
+      <main>
+        <h1 className="page-title">Blog</h1>
+        <div className="blog">
+          {blogs.map(
+            (blog) => (
+              <BlogPreview
+                title={blog.title}
+                date={blog.date}
+                description={blog.description}
+                slug={blog.slug}
```

Comment:
Instead of passing in each prop, you should use the spread operator with `{...blog}`. This will pass in each prop of the JSON object without having to do them individually!